e2e/helpers.py

```diff
@@ -0,0 +1,118 @@
+import boto3
+import os
+import random
+import cis_crypto
+from cis_profile import common
+from cis_profile import profile
+
+
+def get_cis_environment():
+    return os.getenv("CIS_ENVIRONMENT", "development")
+
+
+def get_client_id_path():
+    return "/iam/cis/{}/change_client_id".format(get_cis_environment())
+
+
+def get_client_secret_path():
+    return "/iam/cis/{}/change_service_client_secret".format(get_cis_environment())
+
+
+def get_url_dict():
```

Comment:
This should probably get the URls from the wk endpoint (this tests the wk endpoint as well, which is what people would probably use)

https://auth.allizom.org/.well-known/mozilla-iam in `api.endpoints` and `api.audience`

The well-known lib can get them also very easily